frontend/cypress/tests/chartView/chartView_widget.cy.js

```diff
@@ -0,0 +1,38 @@
+describe("Widget Mode Copy and Logo Verification", () => {
```

Comment:
I think we use full lowercase for test cases and test group names.

```suggestion
describe("widget mode copy and logo verification", () => {
```